frame/staking/src/testing_utils.rs

```diff
@@ -49,11 +49,23 @@ pub fn create_funded_user<T: Config>(
 	n: u32,
 	balance_factor: u32,
 ) -> T::AccountId {
+	create_funded_user_b::<T>(string, n, balance_factor.into())
+}
+
+/// Grab a funded user.
+pub fn create_funded_user_b<T: Config>(
+	string: &'static str,
+	n: u32,
+	balance_factor: crate::BalanceOf<T>,
+) -> T::AccountId {
```

Comment:
I made this just so we could take `BalanceOf<T>` for balance_factor instead of `u32`, because in theory the threshold we select could overflow a `u32`. But it seems strange because now `create_funded_user` just wraps this function (look above). We could replace `create_funded_user` with this and just call `.into` on all the existing `balance_factor` arguments, but that would make this diff a bit more annoying (although we could do a small pr to update master with this). @kianenigma thoughts?